src/legacy/ui/public/vislib/visualizations/gauges/meter.js

```diff
@@ -269,33 +297,10 @@ export function MeterGaugeProvider() {
             if (textTooLong) {
               hiddenLabels = true;
             }
-            return smallContainer || textTooLong ? 'none' : 'initial';
+            return valueLabelHidden || smallContainer || textTooLong ? 'none' : 'initial';
           });
       }
 
-      gauges
```

Comment:
This (value label) has just been moved above the other two labels, so we know whether or not that's shown.